WorkunitExec.ecl

```diff
@@ -522,6 +571,84 @@ EXPORT WorkunitExec := MODULE
             );
 
         RETURN queryResults(rProtected);
-    END;
+    END;  
+
+    /**
+     * Record structure used to represent the results of a published
+     * workunit.
+     */
+     EXPORT PublishResultsLayout := RECORD
+          STRING  wuid {XPATH('Wuid')};
+          STRING  results {XPATH('results')};
+          STRING  QuerySet {XPATH('QuerySet')};
+          STRING  QueryName {XPATH('QueryName')};
+          STRING  QueryId {XPATH('QueryId')};
+      END;
+
+    /**
+     * Finds the latest version of a compiled workunit, by name, and publish it as query.
+     *
+     * @param   jobName             The jobname of the workunit to execute
+     *                              as a string; REQUIRED
+     * @param   espURL              The full URL for accessing the esp process
+     *                              running on the HPCC Systems cluster (this
+     *                              is typically the same URL as used to access
+     *                              ECL Watch); set to an empty string to use
+     *                              the URL of the current esp process;
+     *                              OPTIONAL, defaults to an empty string
+     * @param   clusterName         The name of the cluster in which to look
+     *                              for the compiled workunit with provided
+     *                              jobName;OPTIONAL, defualts to Roxie cluster
```

Comment:
Minor:

- Missing space after semicolon
- "defualts" misspelled
- 'roxie' should have apostrophes and be all lowercase